source/jsonnet/northern-ireland/lib/common_rules.libsonnet

```diff
@@ -63,12 +63,22 @@
     id: 'date-of-birth-answer',
     condition: 'greater than',
     date_comparison: {
-      value: '2020-07-01',
+      value: '2021-07-01',
       offset_by: {
         years: -4,
       },
     },
   },
+  nisraOver5: {
```

Comment:
This name confused me as this is a NISRA specific rules file and we already have an over5 rule. I asked Donna what the significance of it was, she said:

> Anyone aged 4 on 1st July of that year should go to school. If you are aged 4 on 2nd July you have to wait until the next year so shouldn't see any questions on schools.

I think we should rename:
`nisraUnder4` -> `schoolYearUnder4`
`nisraOver5` -> `schoolYearOver5`

Reply:
I've just noticed it's less than or equal to, so it should be something like `schoolYear5OrOver`.

Reply:
Renamed according to your suggestions, thanks👍